src/app/components/Topbar/styles.module.scss

```
@@ -14,6 +15,25 @@ $shadow: 0 0 7px 0 rgba(122, 122, 122, 0.81);
   width: 100%;
 }
 
+.container img {
+  margin-left: auto;
+}
+
 .logo {
   height: calc(#{$topbar-heigth} - 40px);
+  cursor: pointer;
+}
+
+.backButton {
+  font-size: $small * 0.9 !important;
+  background-color: #ff6363 !important;
+  color: $white;
+  padding: 0 10px 0 10px;
+  margin: -8px 0 0 -20px;
+  align-self: start;
```

```suggestion
  align-self: flex-start;
```
has better compatibility at the moment.